cms/admin/forms.py

```diff
@@ -302,14 +302,12 @@ def clean(self):
             # addressed first.
             return data
 
-        parent_node = data.get("parent_node")
-
-        if parent_node:
-            slug = data["slug"]
-            parent_path = parent_node.item.get_path(self._language)
+        if parent_page := data.get('parent_node'):
```

Comment:
Here also, I'd propose to spend one additional line of code for readability:
```suggestion
        parent_page = data.get("parent_node")
        if parent_page:
```